ChristmasTree.java

```diff
@@ -0,0 +1,130 @@
+package LAB7.LAB7;
+
+public class ChristmasTree {
+    public int height = 0;
+    public String producer = "";
+    public float price = 0;
+    public String material = "";
+
+    private String color = "";
+    private int dateMade = 0;
+    private String width = "";
+    private String style = "";
+    private String treeFamily = "";
+
+    private static int DayWhenShouldYouPutUpYourChristmasTree = 29;
+
+    protected int discount = 0;
+    protected String lightBulbType = "";
+
+    public ChristmasTree() {
+    }
+
+    public ChristmasTree(int height, String producer, float price, String material) {
```

Comment:
please call the constructor with a higher count of parameters instead of do the assignments here